packages/theme-compatibility/src/leven/base.scss

```diff
@@ -61,6 +61,7 @@ body {
 	}
 }
 
+.crowdsignal-forms-phone-number-block {
```

Comment:
This line breaks the CSS compilation

Reply:
Also, we might need to increase the selector specificity.
It's still being overridden in the Renderer

Reply:
I've fixed up the CSS. Let me know if there is a better way to do this.

Reply:
I saw you added a closing `}` to the end, but that's wrapping `.crowdsignal-forms-date-picker-block` and `.crowdsignal-forms-time-picker-block` and will break the Date/Time Picker.
To fix it, you need to replace the `.crowdsignal-forms-phone-number-block {` with `.crowdsignal-forms-phone-number-block,` and remove the closing `}` you added last.
In the end, it should look like this:

```CSS
.crowdsignal-forms-phone-number-block,
.crowdsignal-forms-date-picker-block .react-datepicker-wrapper,
.crowdsignal-forms-time-picker-block .react-datepicker-wrapper {
	input {
		background-color: var(--crowdsignal-forms-input-background-color);
		border: 1px solid var(--crowdsignal-forms-input-border-color);
	}
}
```

And regarding my comment to increase the selector specificity:
If you go to the Renderer, the block doesn't have a white background on the Leven theme.
To fix that, change the two `<PhoneInputWrapper>` on `packages/blocks/src/components/form-phone-number/index.js` to:

```JavaScript
<PhoneInputWrapper className="crowdsignal-forms-phone-input-wrapper">
```
And now, change the CSS to:

```CSS
.crowdsignal-forms-phone-number-block .crowdsignal-forms-phone-input-wrapper,
.crowdsignal-forms-date-picker-block .react-datepicker-wrapper,
.crowdsignal-forms-time-picker-block .react-datepicker-wrapper {
	input {
		background-color: var(--crowdsignal-forms-input-background-color);
		border: 1px solid var(--crowdsignal-forms-input-border-color);
	}
}
```

Reply:
Thanks!

The first one was a silly mistake. The linter was complaining about a missing bracket and I didn't notice the extra one there.

And the second point makes total sense. I added it and now works as expected :)